webpack.config.js

```diff
@@ -40,6 +40,17 @@ module.exports = () => {
           use: ['babel-loader'],
         },
 
+        {
+          test: /\.(woff|svg|ttf|woff2|eot)$/,
+          include: path.resolve(__dirname, 'src/fonts'),
```

Comment:
Could you please do not use `/` separator and just replace it with

```
path.resolve(__dirname, 'src', 'fonts')
```

Reply:
fixed